setup.py

```diff
@@ -141,5 +142,12 @@ def run_tests(self):
         'clean': CleanCommand,
         'test': PyTestCommand,
     },
-    install_requires=requires,
+    install_requires=install_requires,
+    setup_requires=[
+        'wheel==0.29.0',
+    ],
+    tests_require=[
```

Comment:
hm, they can be in test_reqs.txt to make travis work, but manually running python setup.py test doesn't work if they aren't also included somewhere here, and tests_require is a more correct place than install_requires.  i'd prefer not to make the user manually pip install -r test_reqs.txt when setuptools provides a mechanism specifically for this purpose.